strings/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ file( GLOB APP_SOURCES RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} *.cpp )
 foreach( testsourcefile ${APP_SOURCES} )
     # I used a simple string replace, to cut off .cpp.
     string( REPLACE ".cpp" "" testname ${testsourcefile} )
-    add_executable( ${testname} ${testsourcefile} )
+    add_executable( ${testname} ${testsourcefile} horspool.cpp)
```

Comment:
No changes are required in the CMake file. Please undo this.

Reply:
Thanks for the advice!